pysensors/utils/__init__.py

```diff
@@ -1,10 +1,25 @@
 from ._base import validate_input
 from ._optimizers import constrained_binary_solve
 from ._optimizers import constrained_multiclass_solve
-
+from ._constraints import get_constraind_sensors_indices
```

Comment:
We will need to add box_constraints and functional_constraints here too

Reply:
Box and Functional constraints are not working yet, that's why I haven't added them. This is our next immediate task.